requirements.txt

```diff
@@ -1,5 +1,10 @@
+attrs==24.2.0
 flake8==5.0.4
 flake8-annotations==2.9.1
 flake8-quotes==3.3.1
 flake8-variables-names==0.0.5
+mccabe==0.7.0
 pep8-naming==0.13.2
+pycodestyle==2.9.1
+pyflakes==2.5.0
+requests==2.32.3
```

Comment:
mccabe, pycodestyle, pyflakes
Did you really need all these packages to solve this homework?
One of the tasks was "The image of the service should be as thin as possible"